lib/common.sh

```diff
@@ -24,17 +24,22 @@ export PROG
 
 set -o errtrace
 
-# OSX not supported.  Tell them right away
+declare -A some_var || (echo "Running with unsupported bash version!" && exit 1)
```

Comment:
should we mention what version is supported?

Reply:
yeah, maybe say "bash 4.0 or greater required"?

Reply:
Done.